packages/itinerary-body/src/styled.tsx

```diff
@@ -85,7 +85,7 @@ export const AnchorButton = styled.a`
   }
 `;
 
-export const LinkButton = styled.a`
+export const StyledLink = styled.a`
```

Comment:
🚨🚨 Breaking changes...

Reply:
I see what is going on. `onClick` on an anchor `<a>` element is incorrect. To use `onClick`, you have to use a button. The correct use with anchors is `href`.

Reply:
This is what I would do as a `fix` to avoid a breaking change:
- Revert the name changes
- Change to `export const LinkButton = styled(TransparentButton)`